scripts/watchlist.js

```diff
@@ -44,14 +33,14 @@ function removeMovieFromLocal(filmID) {
     })
 
     movieWatchlist.splice(foundMovieIndex, 1)
-    console.log(movieWatchlist)
+
     const savedMovies = JSON.stringify(movieWatchlist)
     localStorage.setItem("Watchlist", savedMovies)
-    movieResults = JSON.parse(localStorage.getItem("Watchlist"))
+
     renderWatchlist()
```

Comment:
As `renderWatchlist` fetches the watchlist from localStorage, it's no longer necessary to fetch it in `removeMovieFromLocal`